pyproject.toml

```diff
@@ -0,0 +1,99 @@
+[build-system]
+requires = ["setuptools>=61.0.0"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "django-two-factor-auth"
+version = "1.16.0"
+description = "Complete Two-Factor Authentication for Django"
+readme = "README.rst"
+authors = [
+  {name = "Bouke Haarsma", email = "bouke@haarsma.eu"},
+]
+maintainers = [
+  {name = "Claude Paroz", email = "claude@2xlibre.net"},
+]
```

Comment:
More maintainers can be added.

Reply:
I know I'm not exactly active at the moment, but I'd be happy to have my name here :+1: